content/en/integrations/nxlog.md

```diff
@@ -28,6 +28,10 @@ Configure NXLog to gather logs from your host, containers, & services.
 
 ### Log collection
 
+{{< site-region region="gov,us5,ap1" >}}
```

Comment:
I wanted to make sure you meant to include "gov" here as well, since it's not mentioned in the PR title or ticket. I'm assuming yes, just wanted to verify!